01-eslint-plugin.md

```diff
@@ -0,0 +1,60 @@
+# 01. Set up ESLint to Audit Accessibility Issues in React
+
+**[📹 Video](https://egghead.io/lessons/react-v2-00-a-beginners-guide-to-react-introduction?pl=a-beginners-guide-to-react-v2-6c4d)**
+
+
+Automate accessibility checks in React with an eslint plugin. 👍
```

Comment:
What do you think of starting these lines with the emoji vs the end of the line? My thinking is that if people are scanning for good practices, it's much easier to look at the start of lines than scanning back and forth at the end of a line which can vary

Reply:
Makes sense! 👍